### PR TITLE
Add A11y eslint plugin

### PR DESCRIPTION
**Pull request by LopesAlysson**
# What

Add A11y plugin to make sure we follow the recommended approach on accessibility;

# Why

It's important to make our design system accessible to everyone.

# How

* add `eslint`, `babel-eslint` and `eslint-plugin-jsx-a11y`;
* create `.eslintrc` file;
* run eslint and fix errors;

### CODE REVIEW

**Comment by LopesAlysson** [comment]
[ch20990]

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": ["jsx-a11y"],
+  "extends": ["plugin:jsx-a11y/strict"],
```
Comment (AgtLucas) [inline .eslintrc]: Humm, I wonder if make sense to add such plugin here, as Astro doesn't have any JSX itself, only the docz page...

🤔

Comment (AgtLucas) [reply .eslintrc]: Maybe it's better to move this to `docz` directory.

Comment (LopesAlysson) [reply .eslintrc]: As discussed in our slack channel, we will stick to the root file.

**Pull request merged**